### PR TITLE
EP-32041 : Assessment reports Issues on QA and Preview env

### PR DESCRIPTION
**Pull request by tincklegala**
file type parameter, while transfer is in progress

### CODE REVIEW

**Comment by ShilpeshCoder** [comment]
I think we need to bump up gem version also

**Comment by kapildewade** [comment]
hey guys I forgot to mention earlier, we need to bump up the gem version across multiple brnaches
pls see readme file to bump up the version

**Comment by kapildewade** [comment]
@ShilpeshCoder just saw your comment. pls bump up and run /.reinstall to bump. We've 3 branches used for different versions of rails. If it is too confusion, let me know I'll do it tomorrow morning.

pls run some check locally to see if it is working as it should, for GCP, connect with @vikas-nandanam , need to check through console or app.

**Comment by kapildewade** [comment]
As the changes are needed only in rss collector and one other app, check respective gemfile to see which branch of this repo is used and bump up only those. I would just do all just to avoid conflicts in the future. First make sure it works on master before bumping up on branches.

File: lib/cloud_storage_interface/gcp_gcs_interface.rb [hunk 1]
```diff
@@ -34,7 +34,7 @@ def download_file(bucket_name:, key:, local_path:)
       File.exists?(local_path) # emulating the return val of the S3 API
     end
 
-    def presigned_url(bucket_name:, key:, expires_in:)
+    def presigned_url(bucket_name:, key:, expires_in:, response_content_type:)
       get_object!(bucket_name, key).signed_url(expires: expires_in)
```
Comment (kapildewade) [inline lib/cloud_storage_interface/gcp_gcs_interface.rb]: you've not used `response_content_type:`, pls check gcs ruby docs. It may fail in GCS otherwise.

**Comment by kapildewade** [comment]
I'm hoping you guys won't rush to merge PR without testing it first.

**Comment by tincklegala** [comment]
Tested on gcpstaging.edcastin.com. The assessment report was uploaded successfully in the bucket and sent in downloadable format in email

**Comment by kapildewade** [comment]
if this is working, pls rebase other branches and bump up version there as well

**Pull request merged**